lile.go

```diff
@@ -90,8 +90,8 @@ func GlobalService() *Service {
 func Name(n string) {
 	service.ID = generateID(n)
 	service.Name = n
-	AddUnaryInterceptor(otgrpc.OpenTracingServerInterceptor(
-		fromenv.Tracer(n)))
+	AddUnaryInterceptor(otgrpc.OpenTracingServerInterceptor(fromenv.Tracer(n)))
+	AddUnaryInterceptor(grpc_recovery_logger.UnaryServerInterceptor()) // Added 2nd time to trace
```

Comment:
Can you check how this would affect our sentry tracing? We do want errors there to be captured, but if this interceptor captures them first then they might not be captured by our sentry interceptors.

Reply:
p.s. use the local version of gp_connect!

Reply:
Looking at gp connect the panics will blow through the tracing intercepter in there then the sentry will handle the [recover](https://github.com/echo-health/gp_connect/blob/e7f83c3da1725370d766de1e39ffebeb707630c5/platform/sentry.go#L15) but it doesn't pass back the error which it extracts so I think this is broken as it is. It will return `res nil, err nil` which will have the tracing report that the span was a success and Prometheus report that it was a success too. Sentry however will report an error.

Reply:
https://github.com/echo-health/gp_connect/pull/108/files

this is what's needed in this service I think 👀

Reply:
Okay, in that case, I'd hold off this change until we sort out the sentry stuff.